crawler.js

```diff
@@ -90,6 +91,7 @@ class Crawler {
     this.configureUA();
 
     this.headers = {"User-Agent": this.userAgent};
+    this.pages = [{"format": "json-pages-1.0", "id": "pages", "title": "All Pages", "hasText": false}]
 
```

Comment:
Since we currently allow adding to an existing collection (and probably good to allow, though maybe shouldn't be the default?), this should probably check if `pages/pages.jsonl` already exists, and if it does, we don't output this, and instead append to the pages/pages.jsonl